src/demo/example-data/deckgl-map.json

```diff
@@ -1,110 +1,112 @@
+          "@@type": "ScatterplotLayer",
+          "data": [
+            {
+              "position": [
+                -122.45,
+                37.8
+              ]
+            }
+          ],
+          "getFillColor": [
+            255,
+            0,
+            0,
+            255
+          ],
+          "getRadius": 1000
+        },
+        {
+          "@@type": "TextLayer",
+          "data": [
+            {
+              "position": [
+                -122.45,
+                37.8
+              ],
+              "text": "Hello World"
+            }
+          ]
+        }
+      ]
+    }
+  },
+  {
+    "deckglSpec": {
+      "initialViewState": {
+        "target": [
+          434962.5,
+          6478095.5,
+          0
+        ],
+        "zoom": -3
+      },
+      "layers": [
+        {
+          "@@type": "ColormapLayer",
+          "id": "colormap-layer",
+          "bounds": [
+            432205,
+            6475078,
+            437720,
+            6481113
+          ],
+          "image": "https://i.imgur.com/Dx8eseE.png",
+          "colormap": "https://cdn.jsdelivr.net/gh/kylebarron/deck.gl-raster/assets/colormaps/plasma.png",
+          "valueRange": [
+            2782,
+            3513
+          ],
+          "pickable": true
+        },
+        {
+          "@@type": "Hillshading2DLayer",
+          "id": "hillshading-layer",
+          "bounds": [
+            432205,
+            6475078,
+            437720,
+            6481113
+          ],
+          "valueRange": [
+            2782,
+            3513
+          ],
+          "opacity": 1.0,
+          "image": "https://i.imgur.com/Dx8eseE.png"
+        },
+        {
+          "@@type": "DrawingLayer",
+          "id": "drawing-layer",
+          "mode": "drawLineString",
+          "data": {
+            "type": "FeatureCollection",
+            "features": []
           }
-    },
-    {
-        "deckglSpec": {
-            "initialViewState": {
-                "target": [434962.5, 6478095.5, 0],
-                "zoom": -3
-            },
-            "layers": [
-                {
-                    "@@type": "ColormapLayer",
-                    "id": "colormap-layer",
-                    "bounds": [432205, 6475078, 437720, 6481113],
-                    "image": "https://i.imgur.com/Dx8eseE.png",
-                    "colormap": "https://cdn.jsdelivr.net/gh/kylebarron/deck.gl-raster/assets/colormaps/plasma.png",
-                    "valueRange": [2782, 3513],
-                    "pickable": true
-                },
-                {
-                    "@@type": "Hillshading2DLayer",
-                    "id": "hillshading-layer",
-                    "bounds": [432205, 6475078, 437720, 6481113],
-                    "valueRange": [2782, 3513],
-                    "opacity": 1.0,
-                    "image": "https://i.imgur.com/Dx8eseE.png"
-                },
-                {
-                    "@@type": "WellsLayer",
```

Comment:
The WellsLayer has been removed. Was this intentional?

Reply:
No, I was testing without it and forgot to add it back. I'll add it now.